apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/index.php

```diff
@@ -18,7 +18,7 @@
  */
 function enqueue_launch_button_script_and_style( $site_launch_options ) {
```

Comment:
@yansern 👋🏻 It seems this module is not built as part of the ETK `build` task. I don't see it built in the plugin built from `trunk` either, there's no `dist` directory in the release. Maybe this is not being used anymore?